1_assignment/alphbet_frequency.js

```diff
@@ -1,13 +1,18 @@
 
-string = "u@#d$fkj$#!kds"
 
-function calculateFrequency(string) {
+//Sample String .
+var string = "u@#d$fkj$#!kds";
+
+
+
+// Solution 1 : Using String Functions.
+function calculateFrequencyUsingString(string) {
   // Write your code here
   var i=0;
```

Comment:
Formatting issue. Why not declare inside for statement itself. Use sensible variable names.